features/order/managing_orders/seeing_order_item_taxes.feature

```diff
@@ -25,3 +28,12 @@ Feature: Seeing taxes of order items
         Then the order's shipping total should be "€0.00"
         And the order's tax total should be "€32.20"
         And the order's total should be "€252.20"
+
+    @ui
+    Scenario: Seeing included in price taxes of order items are not counted in taxes total
```

Comment:
Please extract this to a separate feature file.

1. We should not mix included and excluded tax rates within one channel
2. The background becomes too big